app/build.gradle.kts

```diff
@@ -133,3 +134,14 @@ dependencies {
     kspTest(libs.hilt.compiler)
 
 }
+
+dependencyGuard {
+
```

Comment:
Do we need to track all configuration? I would be in favor of only tracking the production build `prodReleaseRuntimeClasspath`.

Reply:
Done